pkg/kubernetes/overrides_test.go

```diff
@@ -130,18 +132,21 @@ func TestApplyDeploymentOverrides(t *testing.T) {
 				},
 				Spec: &v1beta1.DeploymentOverrideSpec{
 					Template: &v1beta1.PodTemplateSpecOverride{
-						Spec: &corev1.PodSpec{
-							Containers: []corev1.Container{
-								{
-									Name: "testcontainer",
-									Resources: corev1.ResourceRequirements{
-										Limits: corev1.ResourceList{
-											corev1.ResourceCPU: apimachineryresource.MustParse("100m"),
-										},
-									},
-								},
-							},
+						Spec: &apiextensionsv1.JSON{
+							Raw: []byte(`{"containers":[{"name":"testcontainer","resources":{"limits":{"cpu":"100m"}}}]}`),
 						},
+						// Spec: &corev1.PodSpec{
```

Comment:
Could you please remove all commented parts ?

Reply:
Done

Reply:
The json was actually generated by GitHub Copilot from the commented code 😊.